rust/private/rust.bzl

```diff
@@ -651,6 +651,12 @@ _rust_test_attrs = {
             Whether to use libtest.
         """),
     ),
+    "_grep_includes": attr.label(
+        allow_single_file = True,
+        cfg = "host",
```

Comment:
@krasimirgg @hlopko just out of curiosity. Should this be `host`? Why not `exec`?

Reply:
Hmm good catch. https://github.com/bazelbuild/rules_rust/pull/1099